modules/caddyhttp/fileserver/staticfiles.go

```diff
@@ -94,6 +94,7 @@ func (fsrv *FileServer) ServeHTTP(w http.ResponseWriter, r *http.Request, _ cadd
 	root := repl.ReplaceAll(fsrv.Root, ".")
 	suffix := repl.ReplaceAll(r.URL.Path, "")
 	filename := sanitizedPathJoin(root, suffix)
+	streamAsZip := r.URL.Query()["zip"] != nil
```

Comment:
I think we should use a more descriptive query parameter name to reduce lieklyhood of being used elsewhere in template.

Perhaps `zipfolder` or `downloadasZip`

Reply:
If possible I don't think we need a query parameter at all; there's already a way to specify how you want your content, and that's through content negotiation. The `Accept` header should be sufficient.

Reply:
I disagree.  I think the concept of a link added into the browse template is very elegant and this would not be possible if using the Accept header.  Perhaps both but I would keep the query param also.

Reply:
Hey there, the accept parameter would not work as a link to be downloaded on the browser unless someone explicitly sends it while requesting a folder like on curl.
About the parameter renaming, I will do it and send later.